mozilla/fx-private-relay
# Count how often the plan comparison matrix elements are interacted with

# New feature description

This merges into #2464 and adds counts of various interactions.

# How to test

Without a tracker blocker and with DoNotTrack disabled, check the console. It should show events that are being sent as the plan matrix elements are scrolled into view and interacted with.

# Checklist
